api/api.go

```diff
@@ -83,20 +83,23 @@ func routes(host, secretKey string, router *mux.Router, dataStore store.DataStor
 	api.router.HandleFunc("/datasets/{id}/editions/{edition}/versions/{version}/dimensions", api.getDimensions).Methods("GET")
```

Comment:
The version PUT should probably also have the publish check on it?

Reply:
It does